src/style.css

```diff
@@ -1,3 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00ACDC;
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+}
+
+.box-item {
+  margin: 20px;
+  display: inline-block;
+  list-style: none;
+}
+
+.box-link {
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+
+.box-item.is-active .box-link,
+.box-item:nth-child(1) .box-link {
+  color: var(--main-color);
+}
+
+.box-item .box-link::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 2px;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  transform: scaleX(0);
+}
+
+.box-item .box-link:hover::after {
```

Comment:
on hover you should make the text blue instead of adding it:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/6456227a-90c6-4b04-b5f4-34ddd682fe84)

`Laptops and computers` link shouldn't be blue always